ooniprobe/Base.lproj/Localizable.strings

```diff
@@ -105,3 +105,7 @@
 "view_data_policy" = "Click here to view our data policy";
 "test_name_finished" = "%@ finished, go to Past Tests to see the results";
 "open_url_alert" = "Do you want to open the following URL?";
+"max_runtime_low" = "The max runtime you have entered is too low, setting to default.";
+"clear_all_tests" = "Clear all tests";
+"clear_all_tests_alert" = "Do you want to remove all tests?";
+"past_tests_empty" = "Past tests restults will appear here once the tests have finished running";
```

Comment:
Ok. I think we should include them at this point in the future release since they are not many strings and it's suboptimal to block on them, but let's ensure that it doesn't break though.

Also there are some typos in the strings.